### PR TITLE
fix: Enable batch processing for version to vulnerability edge creation

### PR DESCRIPTION
**Pull request by yzainee**
Changed the logic from sequential creation of edges between vuln and version node to batch processing

### CODE REVIEW

**Comment by yzainee** [comment]
[test]

File: src/cve.py [hunk 1]
```diff
@@ -212,8 +214,26 @@ def process(self):
...
+                                                    i=vulnerability['id'], o=total_offset))
+                                    call_gremlin(self.prepare_payload
+                                                 (edge_query, edge_bindings))
+                                    edge_bindings['vuln_version'] = []
+                            if num_offset > 0:
+                                total_offset += num_offset
+                                logger.info("Ingesting in batch for "
+                                            "{i}. Offset {o}".format(
+                                                i=vulnerability['id'], o=total_offset))
                                 call_gremlin(self.prepare_payload
```
Comment (arajkumar) [inline src/cve.py:237]: Can this be split into separate function for batch processing?
Similar to https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/v2/stack_aggregator.py#L123:5?

**Review by arajkumar — CHANGES_REQUESTED** [review]

**Comment by yzainee** [comment]
[test]

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**